packages/ewk_webview_flutter/tizen/src/webview.cc

```diff
@@ -0,0 +1,676 @@
+ * Exposes a single method named `postMessage` to JavaScript, which sends a
+ * message over a method channel to the Dart code.
+ */
+void WebView::RegisterJavaScriptChannelName(const std::string& name) {
+  LOG_DEBUG("Register a JavaScript channel: %s", name.c_str());
+  ewk_view_javascript_message_handler_add(
+      webview_instance_, &WebView::OnJavaScriptMessage, name.c_str());
+}
+
+WebView::~WebView() { Dispose(); }
+
+std::string WebView::GetChannelName() {
+  return "plugins.flutter.io/webview_" + std::to_string(GetViewId());
+}
+
+void WebView::Dispose() {
+  evas_object_smart_callback_del(webview_instance_, "offscreen,frame,rendered",
+                                 &WebView::OnFrameRendered);
+  evas_object_smart_callback_del(webview_instance_, "load,started",
+                                 &WebView::OnLoadStarted);
+  evas_object_smart_callback_del(webview_instance_, "load,progress",
+                                 &WebView::OnLoadInProgress);
+  evas_object_smart_callback_del(webview_instance_, "load,finished",
+                                 &WebView::OnLoadFinished);
+  evas_object_smart_callback_del(webview_instance_, "load,error",
+                                 &WebView::OnLoadError);
+  evas_object_smart_callback_del(webview_instance_, "url,changed",
+                                 &WebView::OnUrlChanged);
+  evas_object_smart_callback_del(webview_instance_, "console,message",
+                                 &WebView::OnConsoleMessage);
+  evas_object_smart_callback_del(webview_instance_, "policy,navigation,decide",
+                                 &WebView::OnNavigationPolicy);
+  texture_registrar_->UnregisterTexture(GetTextureId());
+}
+
+void WebView::Resize(double width, double height) {
+  width_ = width;
+  height_ = height;
+
+  if (candidate_surface_) {
+    candidate_surface_ = nullptr;
+  }
+
+  tbm_pool_->Prepare(width_, height_);
+  evas_object_resize(webview_instance_, width_, height_);
+}
+
+void WebView::Touch(int type, int button, double x, double y, double dx,
+                    double dy) {
+  Ewk_Touch_Event_Type mouse_event_type = EWK_TOUCH_START;
+  Evas_Touch_Point_State state = EVAS_TOUCH_POINT_DOWN;
+  if (type == 0) {  // down event
+    mouse_event_type = EWK_TOUCH_START;
+    state = EVAS_TOUCH_POINT_DOWN;
+  } else if (type == 1) {  // move event
+    mouse_event_type = EWK_TOUCH_MOVE;
+    state = EVAS_TOUCH_POINT_MOVE;
+
+  } else if (type == 2) {  // up event
+    mouse_event_type = EWK_TOUCH_END;
+    state = EVAS_TOUCH_POINT_UP;
+  } else {
+    // TODO: Not implemented
+  }
+  Eina_List* pointList = 0;
+  Ewk_Touch_Point* point = new Ewk_Touch_Point;
+  point->id = 0;
+  point->x = x;
+  point->y = y;
+  point->state = state;
+  pointList = eina_list_append(pointList, point);
+
+  ewk_view_feed_touch_event(webview_instance_, mouse_event_type, pointList, 0);
+  eina_list_free(pointList);
+}
+
+bool WebView::SendKey(const char* key, const char* string, const char* compose,
```

Comment:
I got a report about crash with key input. so I tried to fix it. @bwikbs could you review [my fix](https://github.com/bbrto21/plugins/commit/720813f95939fb8fdc19d81a76a13e1027f75acc)?